tests/fixtures/issues/issue-1453.php

```diff
@@ -4042,7 +3989,7 @@
             'MajorVer' => '31',
             'MinorVer' => '0',
             'Platform' => 'MacOSX',
-            'Platform_Version' => '10',
+            'Platform_Version' => '10.11',
```

Comment:
I missed this when I made this test. I look over the tests looking for an "unknown" platform version where there shouldn't be one, but I forgot that OSX typically gets caught as version "10" due to the "OSX" platform.